bundles/ch.elexis.core.ui.mediorder/src/ch/elexis/core/ui/mediorder/MediorderPart.java

```diff
@@ -290,7 +322,7 @@ public String getText(Object element) {
 			}
 		});
 		TableColumn tblclmntvcPatientNumber = tvcPatientNumber.getColumn();
-		tcLayout.setColumnData(tblclmntvcPatientNumber, new ColumnPixelData(70, true, true));
+		tcLayout.setColumnData(tblclmntvcPatientNumber, new ColumnPixelData(100, true, true));
```

Comment:
Hier könntest du generell mal die `ColumnWeightData` anschaun, dann würde das
auch immer die ganze Tabelle in der Breite ausfüllen.